### PR TITLE
Notebook for sensitivity calculation

### PR DESCRIPTION
**Pull request by moralejo**
Calculates sensitivity from a Crab observation (for source-independent and source-dependent analysis)
This was used for the LST-1 performance paper - it was one of the two independent codes with which the sensitivities were computed.

### CODE REVIEW

**Comment by moralejo** [comment]
In case it helps, here is the output as a pdf:

[calculate_sensitivity_from_Crab.pdf](https://github.com/cta-observatory/cta-lstchain/files/12561736/calculate_sensitivity_from_Crab.pdf)

**Comment by moralejo** [comment]
@rlopezcoto @maxnoe @morcuended , any idea why tests fail in test_create_drs4_pedestal_file?

**Comment by morcuended** [comment]
> @rlopezcoto @maxnoe @morcuended , any idea why tests fail in test_create_drs4_pedestal_file?

I think it's due to the new ctapipe_io_lst release v0.22.2, which is now being installed by the CI. The same happens in #1177

File: notebooks/calculate_sensitivity_from_Crab.ipynb [hunk 1]
```diff
@@ -0,0 +1,2032 @@
+{
```
Comment (rlopezcoto) [inline notebooks/calculate_sensitivity_from_Crab.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #17.    # obs_time = 1/60. * u.h # 1 minute
</pre><p>maybe remove all these commented lines or use units like:</p><p>obs_time =&nbsp;1 * u.min</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (rlopezcoto) [inline notebooks/calculate_sensitivity_from_Crab.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #17.    # Three off regions for higher energies:
</pre><p>clarify that this is just to compute the OFF. The sensitivity calculation uses the <code>alpha=0.2</code> defined above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (rlopezcoto) [inline notebooks/calculate_sensitivity_from_Crab.ipynb:1]: <p>you can merge this cell with the one above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (rlopezcoto) [inline notebooks/calculate_sensitivity_from_Crab.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    #%%filprofile
</pre><p>remove?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (rlopezcoto) [inline notebooks/calculate_sensitivity_from_Crab.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.    # SECOND PART OF THE NOTEBOOK!!! RUN FROM HERE OF YOU JUST WANT TO RE-RUN THE CUT OPTIMIZATION  #
</pre><p>typo: OF -&gt; IF</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (rlopezcoto) [inline notebooks/calculate_sensitivity_from_Crab.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #17.    # obs_time = 50 * u.h 
</pre><p>same as above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (moralejo) [reply notebooks/calculate_sensitivity_from_Crab.ipynb:1]: good point, done.

**Pull request merged**